openedx-unsupported/ecommerce
# ENT-1287. Additional coupon usage field for multi times/one customer

[ENT-1287](https://openedx.atlassian.net/browse/ENT-1287) 

**Description:**
This updates the coupon usage field with fourth choice(`Can be used multiple times by one customer`) only for enterprise coupons. 

**Testing Instructions:**
Deployed to Sandbox

0. Enable the `enable_enterprise_offers_for_coupons` switch
1. Goto https://ecommerce-business.sandbox.edx.org/enterprise/coupons/new/ 
2. You should see the `Can be used multiple times by one customer` option in usage dropdown list. This option should not be available for original coupons screen [here](https://ecommerce-business.sandbox.edx.org/coupons/new/)
3. Create an enterprise coupon and it should be created without any issue.
4. You should see the newly created enterprise coupon on coupon list page.
